sources/packages/web/src/components/common/LocationSummary.vue

```diff
@@ -22,13 +23,19 @@
     v-for="item in institutionLocationList"
```

Comment:
mm... its a loop, do we need the same data_cy id for all locations?

Reply:
yeah - the idea is to loop through and use index if we need to get to the specific location